cpp/src/binaryop/compiled/operation.cuh

```diff
@@ -0,0 +1,384 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cudf/column/column_device_view.cuh>
+#include <cudf/types.hpp>
+#include "traits.hpp"
+
+#include <cmath>
+
+#include <cuda/std/type_traits>
+
+using namespace cuda::std;
```

Comment:
Don't do this.

Reply:
This is definitely bad in headers.
I found this used in 
https://github.com/rapidsai/cudf/blob/6db757b7434610f70a5f47c3d2e8d9d1b56e9bca/cpp/src/binaryop/jit/operation.hpp#L31